--- Sphinx-4.0.1/setup.py~	2021-05-10 17:32:03.000000000 +0000
+++ Sphinx-4.0.1/setup.py	2021-05-12 14:45:56.459261770 +0000
@@ -22,7 +22,7 @@
     'sphinxcontrib-serializinghtml',
     'sphinxcontrib-qthelp',
     'Jinja2>=2.3,<3.0',
-    'MarkupSafe<2.0',
+    'MarkupSafe',
     'Pygments>=2.0',
     'docutils>=0.14,<0.18',
     'snowballstemmer>=1.1',
--- Sphinx-4.0.1/Sphinx.egg-info/requires.txt~	2021-05-10 17:33:24.000000000 +0000
+++ Sphinx-4.0.1/Sphinx.egg-info/requires.txt	2021-05-12 14:46:17.027261483 +0000
@@ -5,7 +5,7 @@
 sphinxcontrib-serializinghtml
 sphinxcontrib-qthelp
 Jinja2<3.0,>=2.3
-MarkupSafe<2.0
+MarkupSafe
 Pygments>=2.0
 docutils<0.18,>=0.14
 snowballstemmer>=1.1
